### DISCUSSION TITLE
Provide method to hide all lines on a history chart

### DISCUSSION THREAD

**Discussion by Troon** [home-assistant/frontend#11279] [Other feature requests]
## The request

In the line graphs on the history panel, individual lines (series) can be hidden by clicking on the item in the legend. This strikes through the legend entry and removes the line. Like this:

![image](https://user-images.githubusercontent.com/1823375/87782782-0d450f00-c82b-11ea-9efa-135b94ca2aba.png)

I would like a feature to hide all of the lines in one click, so that I can add in just the ones I want. I'd suggest a "[hide all]" link at the beginning of the legend.

## The alternatives

Currently, the only way to view and compare a select few data series is to go through and click all the unwanted ones. With a lot of sensors (particularly % and temperature) this is a long process.

**Comment by Troon** [comment]
Happy to have a go at this myself,  but I'd need a pointer as to where to start looking

**Comment by DanielBaulig** [comment]
Would like to see this too. Here's probably a good spot to start:

https://github.com/home-assistant/frontend/blob/0a92c28bac80a0bddc3f54577fc62670f3974c15/src/components/entity/ha-chart-base.js

**Comment by katonage** [comment]
Is it possible to have something similar for the state stripes too? Now there is an entity selector there to select one entity to show. It would be nice to have the possibility to select a set of entities and retain this selection over sessions.

**Comment by Youpimatin** [comment]
I didn't see your issue at the time, and I wrote :
**Unselect all entities with a single click (History graph)** #7392

**Comment by bradyjoh** [comment]
Still awaiting a solution, I posted an update in https://github.com/home-assistant/frontend/discussions/7392
